packages/blast-utilities/delay.js

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
In my opinion it is  better to have file such as "blast-helper" (or some better name) to contain small and useful generic functions like delay instead of separate file.

Reply:
I don't think that we have a generic utils file. 
Lets stick to blast-helper.js?